Userland/Libraries/LibGL/CMakeLists.txt

```diff
@@ -20,6 +20,9 @@ set(SOURCES
     Vertex.cpp
 )
 
+# FIXME: Figure out what is going wrong here
```

Comment:
Can you make this FIXME more descriptive or explain it in the commit message?

Reply:
Added some notes, but if you feel adventurous have a trace back

<details>
<summary> Build log </summary>


```
[0/2] Re-checking globbed directories...
[1/11] Building CXX object Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Vertex.cpp.o
FAILED: Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Vertex.cpp.o 
/usr/bin/ccache /usr/bin/clang++ -DLibGL_EXPORTS -I/home/leon/serenity/Meta/Lagom/../.. -I/home/leon/serenity/Meta/Lagom/../../Userland -I/home/leon/serenity/Meta/Lagom/../../Userland/Libraries -I/home/leon/serenity/Meta/Lagom/../../Userland/Services -I/home/leon/serenity/Build/lagom -I/home/leon/serenity/Build/lagom/Userland/Libraries -I/home/leon/serenity/Build/lagom/Userland/Services -fPIC -Wall -Wextra -Werror -Wno-implicit-const-int-float-conversion -Wno-literal-suffix -Wno-maybe-uninitialized -Wno-unknown-warning-option -Wno-unused-command-line-argument -Wnogcc-compat -fsigned-char -fno-exceptions -fdiagnostics-color=always -g1 -O2 -fno-semantic-interposition -fPIC -Wno-overloaded-virtual -Wno-user-defined-literals -fconstexpr-steps=16777216 -Wno-unused-private-field -std=c++20 -MD -MT Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Vertex.cpp.o -MF Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Vertex.cpp.o.d -o Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Vertex.cpp.o -c /home/leon/serenity/Userland/Libraries/LibGL/Vertex.cpp
In file included from /home/leon/serenity/Userland/Libraries/LibGL/Vertex.cpp:10:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/NumericLimits.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Types.h:10:
/home/leon/serenity/Meta/Lagom/../../AK/StdLibExtras.h:75:12: error: cannot cast from lvalue of type 'const Gfx::Matrix<4, float>' to rvalue reference type 'Gfx::Matrix<4, float> &&'; types are not compatible
    return static_cast<T&&>(arg);
           ^~~~~~~~~~~~~~~~~~~~~
/home/leon/serenity/Meta/Lagom/../../AK/Tuple.h:184:32: note: in instantiation of function template specialization 'std::move<Gfx::Matrix<4, float>>' requested here
        : Detail::Tuple<Ts...>(move(other.get<Is>())...)
                               ^
/home/leon/serenity/Meta/Lagom/../../AK/Tuple.h:122:11: note: in instantiation of function template specialization 'AK::Tuple<const Gfx::Matrix<4, float> &>::Tuple<0U>' requested here
        : Tuple(move(other), Indices())
          ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:60:28: note: in instantiation of member function 'AK::Tuple<const Gfx::Matrix<4, float> &>::Tuple' requested here
            new (new_data) F(move(*bit_cast<F*>(old_data)));
                           ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: in instantiation of member function 'AK::Detail::Variant<unsigned char, '\t', AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<unsigned int>, AK::Tuple<double, double, double, double, double, double>, AK::Tuple<>, AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<float, float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<int, unsigned int, const void *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned char>, AK::Tuple<int, int, unsigned int, unsigned int, const void *>, AK::Tuple<double, double>, AK::Tuple<float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int, unsigned int, int, unsigned int>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, unsigned int, unsigned int>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<float>, AK::Tuple<unsigned int>, AK::Tuple<>, AK::Tuple<unsigned int, float, float, float, float>, AK::Tuple<int, int, float, float, float, float, const unsigned char *>, AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>, AK::Tuple<double, double, double, double>, AK::Tuple<unsigned int, unsigned int, Gfx::VectorN<4, float>>, AK::Tuple<unsigned int, unsigned int, int>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, const float *>, AK::Tuple<unsigned int, int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, void *, unsigned int>, AK::Tuple<unsigned int, const double *>, AK::Tuple<unsigned int, int, int, int, int, int, int, int>, AK::Tuple<float>>::move_' requested here
            Variant<IndexType, InitialIndex + 1, Ts...>::move_(old_id, old_data, new_data);
                                                         ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: in instantiation of member function 'AK::Detail::Variant<unsigned char, '\b', AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<unsigned int>, AK::Tuple<double, double, double, double, double, double>, AK::Tuple<>, AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<float, float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<int, unsigned int, const void *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned char>, AK::Tuple<int, int, unsigned int, unsigned int, const void *>, AK::Tuple<double, double>, AK::Tuple<float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int, unsigned int, int, unsigned int>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, unsigned int, unsigned int>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<float>, AK::Tuple<unsigned int>, AK::Tuple<>, AK::Tuple<unsigned int, float, float, float, float>, AK::Tuple<int, int, float, float, float, float, const unsigned char *>, AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>, AK::Tuple<double, double, double, double>, AK::Tuple<unsigned int, unsigned int, Gfx::VectorN<4, float>>, AK::Tuple<unsigned int, unsigned int, int>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, const float *>, AK::Tuple<unsigned int, int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, void *, unsigned int>, AK::Tuple<unsigned int, const double *>, AK::Tuple<unsigned int, int, int, int, int, int, int, int>, AK::Tuple<float>>::move_' requested here
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: (skipping 9 contexts in backtrace; use -ftemplate-backtrace-limit=0 to see all)
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:641:16: note: in instantiation of member function 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_ensure_capacity' requested here
        return try_ensure_capacity(padded_capacity(needed_capacity));
               ^
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:589:13: note: in instantiation of member function 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_grow_capacity' requested here
        TRY(try_grow_capacity(m_size + 1));
            ^
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:304:14: note: in instantiation of function template specialization 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_empend<void (GL::GLContext::*)(float, float, float, float), AK::Tuple<float, float, float, float>>' requested here
        MUST(try_empend(forward<Args>(args)...));
             ^
/home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:267:50: note: in instantiation of function template specialization 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::empend<void (GL::GLContext::*)(float, float, float, float), AK::Tuple<float, float, float, float>>' requested here
        m_current_listing_index->listing.entries.empend(member, Listing::ArgumentsFor<member> { forward<Args>(args)... });
                                                 ^
/home/leon/serenity/Userland/Libraries/LibGL/Vertex.cpp:94:5: note: in instantiation of function template specialization 'GL::GLContext::append_to_listing<&GL::GLContext::gl_color, float &, float &, float &, float &>' requested here
    APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED(gl_color, r, g, b, a);
    ^
/home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:40:9: note: expanded from macro 'APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED'
        append_to_listing<&GLContext::name>(__VA_ARGS__);   \
        ^
1 error generated.
[2/11] Building CXX object Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/ClipPlane.cpp.o
FAILED: Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/ClipPlane.cpp.o 
/usr/bin/ccache /usr/bin/clang++ -DLibGL_EXPORTS -I/home/leon/serenity/Meta/Lagom/../.. -I/home/leon/serenity/Meta/Lagom/../../Userland -I/home/leon/serenity/Meta/Lagom/../../Userland/Libraries -I/home/leon/serenity/Meta/Lagom/../../Userland/Services -I/home/leon/serenity/Build/lagom -I/home/leon/serenity/Build/lagom/Userland/Libraries -I/home/leon/serenity/Build/lagom/Userland/Services -fPIC -Wall -Wextra -Werror -Wno-implicit-const-int-float-conversion -Wno-literal-suffix -Wno-maybe-uninitialized -Wno-unknown-warning-option -Wno-unused-command-line-argument -Wnogcc-compat -fsigned-char -fno-exceptions -fdiagnostics-color=always -g1 -O2 -fno-semantic-interposition -fPIC -Wno-overloaded-virtual -Wno-user-defined-literals -fconstexpr-steps=16777216 -Wno-unused-private-field -std=c++20 -MD -MT Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/ClipPlane.cpp.o -MF Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/ClipPlane.cpp.o.d -o Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/ClipPlane.cpp.o -c /home/leon/serenity/Userland/Libraries/LibGL/ClipPlane.cpp
In file included from /home/leon/serenity/Userland/Libraries/LibGL/ClipPlane.cpp:10:
In file included from /home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:11:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/ByteBuffer.h:11:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Error.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/StringView.h:10:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Checked.h:31:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Concepts.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Forward.h:10:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/SinglyLinkedListSizePolicy.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Types.h:10:
/home/leon/serenity/Meta/Lagom/../../AK/StdLibExtras.h:75:12: error: cannot cast from lvalue of type 'const Gfx::Matrix<4, float>' to rvalue reference type 'Gfx::Matrix<4, float> &&'; types are not compatible
    return static_cast<T&&>(arg);
           ^~~~~~~~~~~~~~~~~~~~~
/home/leon/serenity/Meta/Lagom/../../AK/Tuple.h:184:32: note: in instantiation of function template specialization 'std::move<Gfx::Matrix<4, float>>' requested here
        : Detail::Tuple<Ts...>(move(other.get<Is>())...)
                               ^
/home/leon/serenity/Meta/Lagom/../../AK/Tuple.h:122:11: note: in instantiation of function template specialization 'AK::Tuple<const Gfx::Matrix<4, float> &>::Tuple<0U>' requested here
        : Tuple(move(other), Indices())
          ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:60:28: note: in instantiation of member function 'AK::Tuple<const Gfx::Matrix<4, float> &>::Tuple' requested here
            new (new_data) F(move(*bit_cast<F*>(old_data)));
                           ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: in instantiation of member function 'AK::Detail::Variant<unsigned char, '\t', AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<unsigned int>, AK::Tuple<double, double, double, double, double, double>, AK::Tuple<>, AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<float, float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<int, unsigned int, const void *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned char>, AK::Tuple<int, int, unsigned int, unsigned int, const void *>, AK::Tuple<double, double>, AK::Tuple<float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int, unsigned int, int, unsigned int>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, unsigned int, unsigned int>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<float>, AK::Tuple<unsigned int>, AK::Tuple<>, AK::Tuple<unsigned int, float, float, float, float>, AK::Tuple<int, int, float, float, float, float, const unsigned char *>, AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>, AK::Tuple<double, double, double, double>, AK::Tuple<unsigned int, unsigned int, Gfx::VectorN<4, float>>, AK::Tuple<unsigned int, unsigned int, int>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, const float *>, AK::Tuple<unsigned int, int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, void *, unsigned int>, AK::Tuple<unsigned int, const double *>, AK::Tuple<unsigned int, int, int, int, int, int, int, int>, AK::Tuple<float>>::move_' requested here
            Variant<IndexType, InitialIndex + 1, Ts...>::move_(old_id, old_data, new_data);
                                                         ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: in instantiation of member function 'AK::Detail::Variant<unsigned char, '\b', AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<unsigned int>, AK::Tuple<double, double, double, double, double, double>, AK::Tuple<>, AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<float, float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<int, unsigned int, const void *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned char>, AK::Tuple<int, int, unsigned int, unsigned int, const void *>, AK::Tuple<double, double>, AK::Tuple<float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int, unsigned int, int, unsigned int>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, unsigned int, unsigned int>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<float>, AK::Tuple<unsigned int>, AK::Tuple<>, AK::Tuple<unsigned int, float, float, float, float>, AK::Tuple<int, int, float, float, float, float, const unsigned char *>, AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>, AK::Tuple<double, double, double, double>, AK::Tuple<unsigned int, unsigned int, Gfx::VectorN<4, float>>, AK::Tuple<unsigned int, unsigned int, int>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, const float *>, AK::Tuple<unsigned int, int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, void *, unsigned int>, AK::Tuple<unsigned int, const double *>, AK::Tuple<unsigned int, int, int, int, int, int, int, int>, AK::Tuple<float>>::move_' requested here
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: (skipping 9 contexts in backtrace; use -ftemplate-backtrace-limit=0 to see all)
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:641:16: note: in instantiation of member function 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_ensure_capacity' requested here
        return try_ensure_capacity(padded_capacity(needed_capacity));
               ^
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:589:13: note: in instantiation of member function 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_grow_capacity' requested here
        TRY(try_grow_capacity(m_size + 1));
            ^
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:304:14: note: in instantiation of function template specialization 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_empend<void (GL::GLContext::*)(unsigned int, const double *), AK::Tuple<unsigned int, const double *>>' requested here
        MUST(try_empend(forward<Args>(args)...));
             ^
/home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:267:50: note: in instantiation of function template specialization 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::empend<void (GL::GLContext::*)(unsigned int, const double *), AK::Tuple<unsigned int, const double *>>' requested here
        m_current_listing_index->listing.entries.empend(member, Listing::ArgumentsFor<member> { forward<Args>(args)... });
                                                 ^
/home/leon/serenity/Userland/Libraries/LibGL/ClipPlane.cpp:16:5: note: in instantiation of function template specialization 'GL::GLContext::append_to_listing<&GL::GLContext::gl_clip_plane, unsigned int &, const double *&>' requested here
    APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED(gl_clip_plane, plane, equation);
    ^
/home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:40:9: note: expanded from macro 'APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED'
        append_to_listing<&GLContext::name>(__VA_ARGS__);   \
        ^
1 error generated.
[3/11] Building CXX object Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/ContextParameter.cpp.o
FAILED: Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/ContextParameter.cpp.o 
/usr/bin/ccache /usr/bin/clang++ -DLibGL_EXPORTS -I/home/leon/serenity/Meta/Lagom/../.. -I/home/leon/serenity/Meta/Lagom/../../Userland -I/home/leon/serenity/Meta/Lagom/../../Userland/Libraries -I/home/leon/serenity/Meta/Lagom/../../Userland/Services -I/home/leon/serenity/Build/lagom -I/home/leon/serenity/Build/lagom/Userland/Libraries -I/home/leon/serenity/Build/lagom/Userland/Services -fPIC -Wall -Wextra -Werror -Wno-implicit-const-int-float-conversion -Wno-literal-suffix -Wno-maybe-uninitialized -Wno-unknown-warning-option -Wno-unused-command-line-argument -Wnogcc-compat -fsigned-char -fno-exceptions -fdiagnostics-color=always -g1 -O2 -fno-semantic-interposition -fPIC -Wno-overloaded-virtual -Wno-user-defined-literals -fconstexpr-steps=16777216 -Wno-unused-private-field -std=c++20 -MD -MT Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/ContextParameter.cpp.o -MF Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/ContextParameter.cpp.o.d -o Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/ContextParameter.cpp.o -c /home/leon/serenity/Userland/Libraries/LibGL/ContextParameter.cpp
In file included from /home/leon/serenity/Userland/Libraries/LibGL/ContextParameter.cpp:10:
In file included from /home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:11:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/ByteBuffer.h:11:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Error.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/StringView.h:10:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Checked.h:31:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Concepts.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Forward.h:10:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/SinglyLinkedListSizePolicy.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Types.h:10:
/home/leon/serenity/Meta/Lagom/../../AK/StdLibExtras.h:75:12: error: cannot cast from lvalue of type 'const Gfx::Matrix<4, float>' to rvalue reference type 'Gfx::Matrix<4, float> &&'; types are not compatible
    return static_cast<T&&>(arg);
           ^~~~~~~~~~~~~~~~~~~~~
/home/leon/serenity/Meta/Lagom/../../AK/Tuple.h:184:32: note: in instantiation of function template specialization 'std::move<Gfx::Matrix<4, float>>' requested here
        : Detail::Tuple<Ts...>(move(other.get<Is>())...)
                               ^
/home/leon/serenity/Meta/Lagom/../../AK/Tuple.h:122:11: note: in instantiation of function template specialization 'AK::Tuple<const Gfx::Matrix<4, float> &>::Tuple<0U>' requested here
        : Tuple(move(other), Indices())
          ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:60:28: note: in instantiation of member function 'AK::Tuple<const Gfx::Matrix<4, float> &>::Tuple' requested here
            new (new_data) F(move(*bit_cast<F*>(old_data)));
                           ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: in instantiation of member function 'AK::Detail::Variant<unsigned char, '\t', AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<unsigned int>, AK::Tuple<double, double, double, double, double, double>, AK::Tuple<>, AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<float, float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<int, unsigned int, const void *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned char>, AK::Tuple<int, int, unsigned int, unsigned int, const void *>, AK::Tuple<double, double>, AK::Tuple<float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int, unsigned int, int, unsigned int>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, unsigned int, unsigned int>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<float>, AK::Tuple<unsigned int>, AK::Tuple<>, AK::Tuple<unsigned int, float, float, float, float>, AK::Tuple<int, int, float, float, float, float, const unsigned char *>, AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>, AK::Tuple<double, double, double, double>, AK::Tuple<unsigned int, unsigned int, Gfx::VectorN<4, float>>, AK::Tuple<unsigned int, unsigned int, int>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, const float *>, AK::Tuple<unsigned int, int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, void *, unsigned int>, AK::Tuple<unsigned int, const double *>, AK::Tuple<unsigned int, int, int, int, int, int, int, int>, AK::Tuple<float>>::move_' requested here
            Variant<IndexType, InitialIndex + 1, Ts...>::move_(old_id, old_data, new_data);
                                                         ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: in instantiation of member function 'AK::Detail::Variant<unsigned char, '\b', AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<unsigned int>, AK::Tuple<double, double, double, double, double, double>, AK::Tuple<>, AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<float, float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<int, unsigned int, const void *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned char>, AK::Tuple<int, int, unsigned int, unsigned int, const void *>, AK::Tuple<double, double>, AK::Tuple<float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int, unsigned int, int, unsigned int>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, unsigned int, unsigned int>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<float>, AK::Tuple<unsigned int>, AK::Tuple<>, AK::Tuple<unsigned int, float, float, float, float>, AK::Tuple<int, int, float, float, float, float, const unsigned char *>, AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>, AK::Tuple<double, double, double, double>, AK::Tuple<unsigned int, unsigned int, Gfx::VectorN<4, float>>, AK::Tuple<unsigned int, unsigned int, int>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, const float *>, AK::Tuple<unsigned int, int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, void *, unsigned int>, AK::Tuple<unsigned int, const double *>, AK::Tuple<unsigned int, int, int, int, int, int, int, int>, AK::Tuple<float>>::move_' requested here
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: (skipping 9 contexts in backtrace; use -ftemplate-backtrace-limit=0 to see all)
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:641:16: note: in instantiation of member function 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_ensure_capacity' requested here
        return try_ensure_capacity(padded_capacity(needed_capacity));
               ^
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:589:13: note: in instantiation of member function 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_grow_capacity' requested here
        TRY(try_grow_capacity(m_size + 1));
            ^
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:304:14: note: in instantiation of function template specialization 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_empend<void (GL::GLContext::*)(unsigned int), AK::Tuple<unsigned int>>' requested here
        MUST(try_empend(forward<Args>(args)...));
             ^
/home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:267:50: note: in instantiation of function template specialization 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::empend<void (GL::GLContext::*)(unsigned int), AK::Tuple<unsigned int>>' requested here
        m_current_listing_index->listing.entries.empend(member, Listing::ArgumentsFor<member> { forward<Args>(args)... });
                                                 ^
/home/leon/serenity/Userland/Libraries/LibGL/ContextParameter.cpp:210:5: note: in instantiation of function template specialization 'GL::GLContext::append_to_listing<&GL::GLContext::gl_disable, unsigned int &>' requested here
    APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED(gl_disable, capability);
    ^
/home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:40:9: note: expanded from macro 'APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED'
        append_to_listing<&GLContext::name>(__VA_ARGS__);   \
        ^
1 error generated.
[4/11] Building CXX object Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Stencil.cpp.o
FAILED: Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Stencil.cpp.o 
/usr/bin/ccache /usr/bin/clang++ -DLibGL_EXPORTS -I/home/leon/serenity/Meta/Lagom/../.. -I/home/leon/serenity/Meta/Lagom/../../Userland -I/home/leon/serenity/Meta/Lagom/../../Userland/Libraries -I/home/leon/serenity/Meta/Lagom/../../Userland/Services -I/home/leon/serenity/Build/lagom -I/home/leon/serenity/Build/lagom/Userland/Libraries -I/home/leon/serenity/Build/lagom/Userland/Services -fPIC -Wall -Wextra -Werror -Wno-implicit-const-int-float-conversion -Wno-literal-suffix -Wno-maybe-uninitialized -Wno-unknown-warning-option -Wno-unused-command-line-argument -Wnogcc-compat -fsigned-char -fno-exceptions -fdiagnostics-color=always -g1 -O2 -fno-semantic-interposition -fPIC -Wno-overloaded-virtual -Wno-user-defined-literals -fconstexpr-steps=16777216 -Wno-unused-private-field -std=c++20 -MD -MT Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Stencil.cpp.o -MF Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Stencil.cpp.o.d -o Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Stencil.cpp.o -c /home/leon/serenity/Userland/Libraries/LibGL/Stencil.cpp
In file included from /home/leon/serenity/Userland/Libraries/LibGL/Stencil.cpp:9:
In file included from /home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:11:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/ByteBuffer.h:11:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Error.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/StringView.h:10:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Checked.h:31:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Concepts.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Forward.h:10:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/SinglyLinkedListSizePolicy.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Types.h:10:
/home/leon/serenity/Meta/Lagom/../../AK/StdLibExtras.h:75:12: error: cannot cast from lvalue of type 'const Gfx::Matrix<4, float>' to rvalue reference type 'Gfx::Matrix<4, float> &&'; types are not compatible
    return static_cast<T&&>(arg);
           ^~~~~~~~~~~~~~~~~~~~~
/home/leon/serenity/Meta/Lagom/../../AK/Tuple.h:184:32: note: in instantiation of function template specialization 'std::move<Gfx::Matrix<4, float>>' requested here
        : Detail::Tuple<Ts...>(move(other.get<Is>())...)
                               ^
/home/leon/serenity/Meta/Lagom/../../AK/Tuple.h:122:11: note: in instantiation of function template specialization 'AK::Tuple<const Gfx::Matrix<4, float> &>::Tuple<0U>' requested here
        : Tuple(move(other), Indices())
          ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:60:28: note: in instantiation of member function 'AK::Tuple<const Gfx::Matrix<4, float> &>::Tuple' requested here
            new (new_data) F(move(*bit_cast<F*>(old_data)));
                           ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: in instantiation of member function 'AK::Detail::Variant<unsigned char, '\t', AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<unsigned int>, AK::Tuple<double, double, double, double, double, double>, AK::Tuple<>, AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<float, float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<int, unsigned int, const void *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned char>, AK::Tuple<int, int, unsigned int, unsigned int, const void *>, AK::Tuple<double, double>, AK::Tuple<float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int, unsigned int, int, unsigned int>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, unsigned int, unsigned int>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<float>, AK::Tuple<unsigned int>, AK::Tuple<>, AK::Tuple<unsigned int, float, float, float, float>, AK::Tuple<int, int, float, float, float, float, const unsigned char *>, AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>, AK::Tuple<double, double, double, double>, AK::Tuple<unsigned int, unsigned int, Gfx::VectorN<4, float>>, AK::Tuple<unsigned int, unsigned int, int>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, const float *>, AK::Tuple<unsigned int, int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, void *, unsigned int>, AK::Tuple<unsigned int, const double *>, AK::Tuple<unsigned int, int, int, int, int, int, int, int>, AK::Tuple<float>>::move_' requested here
            Variant<IndexType, InitialIndex + 1, Ts...>::move_(old_id, old_data, new_data);
                                                         ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: in instantiation of member function 'AK::Detail::Variant<unsigned char, '\b', AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<unsigned int>, AK::Tuple<double, double, double, double, double, double>, AK::Tuple<>, AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<float, float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<int, unsigned int, const void *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned char>, AK::Tuple<int, int, unsigned int, unsigned int, const void *>, AK::Tuple<double, double>, AK::Tuple<float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int, unsigned int, int, unsigned int>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, unsigned int, unsigned int>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<float>, AK::Tuple<unsigned int>, AK::Tuple<>, AK::Tuple<unsigned int, float, float, float, float>, AK::Tuple<int, int, float, float, float, float, const unsigned char *>, AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>, AK::Tuple<double, double, double, double>, AK::Tuple<unsigned int, unsigned int, Gfx::VectorN<4, float>>, AK::Tuple<unsigned int, unsigned int, int>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, const float *>, AK::Tuple<unsigned int, int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, void *, unsigned int>, AK::Tuple<unsigned int, const double *>, AK::Tuple<unsigned int, int, int, int, int, int, int, int>, AK::Tuple<float>>::move_' requested here
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: (skipping 9 contexts in backtrace; use -ftemplate-backtrace-limit=0 to see all)
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:641:16: note: in instantiation of member function 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_ensure_capacity' requested here
        return try_ensure_capacity(padded_capacity(needed_capacity));
               ^
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:589:13: note: in instantiation of member function 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_grow_capacity' requested here
        TRY(try_grow_capacity(m_size + 1));
            ^
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:304:14: note: in instantiation of function template specialization 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_empend<void (GL::GLContext::*)(int), AK::Tuple<int>>' requested here
        MUST(try_empend(forward<Args>(args)...));
             ^
/home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:267:50: note: in instantiation of function template specialization 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::empend<void (GL::GLContext::*)(int), AK::Tuple<int>>' requested here
        m_current_listing_index->listing.entries.empend(member, Listing::ArgumentsFor<member> { forward<Args>(args)... });
                                                 ^
/home/leon/serenity/Userland/Libraries/LibGL/Stencil.cpp:15:5: note: in instantiation of function template specialization 'GL::GLContext::append_to_listing<&GL::GLContext::gl_clear_stencil, int &>' requested here
    APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED(gl_clear_stencil, s);
    ^
/home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:40:9: note: expanded from macro 'APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED'
        append_to_listing<&GLContext::name>(__VA_ARGS__);   \
        ^
1 error generated.
[5/11] Building CXX object Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Lighting.cpp.o
FAILED: Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Lighting.cpp.o 
/usr/bin/ccache /usr/bin/clang++ -DLibGL_EXPORTS -I/home/leon/serenity/Meta/Lagom/../.. -I/home/leon/serenity/Meta/Lagom/../../Userland -I/home/leon/serenity/Meta/Lagom/../../Userland/Libraries -I/home/leon/serenity/Meta/Lagom/../../Userland/Services -I/home/leon/serenity/Build/lagom -I/home/leon/serenity/Build/lagom/Userland/Libraries -I/home/leon/serenity/Build/lagom/Userland/Services -fPIC -Wall -Wextra -Werror -Wno-implicit-const-int-float-conversion -Wno-literal-suffix -Wno-maybe-uninitialized -Wno-unknown-warning-option -Wno-unused-command-line-argument -Wnogcc-compat -fsigned-char -fno-exceptions -fdiagnostics-color=always -g1 -O2 -fno-semantic-interposition -fPIC -Wno-overloaded-virtual -Wno-user-defined-literals -fconstexpr-steps=16777216 -Wno-unused-private-field -std=c++20 -MD -MT Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Lighting.cpp.o -MF Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Lighting.cpp.o.d -o Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Lighting.cpp.o -c /home/leon/serenity/Userland/Libraries/LibGL/Lighting.cpp
In file included from /home/leon/serenity/Userland/Libraries/LibGL/Lighting.cpp:9:
In file included from /home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:11:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/ByteBuffer.h:11:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Error.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/StringView.h:10:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Checked.h:31:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Concepts.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Forward.h:10:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/SinglyLinkedListSizePolicy.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Types.h:10:
/home/leon/serenity/Meta/Lagom/../../AK/StdLibExtras.h:75:12: error: cannot cast from lvalue of type 'const Gfx::Matrix<4, float>' to rvalue reference type 'Gfx::Matrix<4, float> &&'; types are not compatible
    return static_cast<T&&>(arg);
           ^~~~~~~~~~~~~~~~~~~~~
/home/leon/serenity/Meta/Lagom/../../AK/Tuple.h:184:32: note: in instantiation of function template specialization 'std::move<Gfx::Matrix<4, float>>' requested here
        : Detail::Tuple<Ts...>(move(other.get<Is>())...)
                               ^
/home/leon/serenity/Meta/Lagom/../../AK/Tuple.h:122:11: note: in instantiation of function template specialization 'AK::Tuple<const Gfx::Matrix<4, float> &>::Tuple<0U>' requested here
        : Tuple(move(other), Indices())
          ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:60:28: note: in instantiation of member function 'AK::Tuple<const Gfx::Matrix<4, float> &>::Tuple' requested here
            new (new_data) F(move(*bit_cast<F*>(old_data)));
                           ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: in instantiation of member function 'AK::Detail::Variant<unsigned char, '\t', AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<unsigned int>, AK::Tuple<double, double, double, double, double, double>, AK::Tuple<>, AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<float, float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<int, unsigned int, const void *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned char>, AK::Tuple<int, int, unsigned int, unsigned int, const void *>, AK::Tuple<double, double>, AK::Tuple<float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int, unsigned int, int, unsigned int>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, unsigned int, unsigned int>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<float>, AK::Tuple<unsigned int>, AK::Tuple<>, AK::Tuple<unsigned int, float, float, float, float>, AK::Tuple<int, int, float, float, float, float, const unsigned char *>, AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>, AK::Tuple<double, double, double, double>, AK::Tuple<unsigned int, unsigned int, Gfx::VectorN<4, float>>, AK::Tuple<unsigned int, unsigned int, int>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, const float *>, AK::Tuple<unsigned int, int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, void *, unsigned int>, AK::Tuple<unsigned int, const double *>, AK::Tuple<unsigned int, int, int, int, int, int, int, int>, AK::Tuple<float>>::move_' requested here
            Variant<IndexType, InitialIndex + 1, Ts...>::move_(old_id, old_data, new_data);
                                                         ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: in instantiation of member function 'AK::Detail::Variant<unsigned char, '\b', AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<unsigned int>, AK::Tuple<double, double, double, double, double, double>, AK::Tuple<>, AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<float, float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<int, unsigned int, const void *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned char>, AK::Tuple<int, int, unsigned int, unsigned int, const void *>, AK::Tuple<double, double>, AK::Tuple<float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int, unsigned int, int, unsigned int>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, unsigned int, unsigned int>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<float>, AK::Tuple<unsigned int>, AK::Tuple<>, AK::Tuple<unsigned int, float, float, float, float>, AK::Tuple<int, int, float, float, float, float, const unsigned char *>, AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>, AK::Tuple<double, double, double, double>, AK::Tuple<unsigned int, unsigned int, Gfx::VectorN<4, float>>, AK::Tuple<unsigned int, unsigned int, int>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, const float *>, AK::Tuple<unsigned int, int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, void *, unsigned int>, AK::Tuple<unsigned int, const double *>, AK::Tuple<unsigned int, int, int, int, int, int, int, int>, AK::Tuple<float>>::move_' requested here
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: (skipping 9 contexts in backtrace; use -ftemplate-backtrace-limit=0 to see all)
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:641:16: note: in instantiation of member function 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_ensure_capacity' requested here
        return try_ensure_capacity(padded_capacity(needed_capacity));
               ^
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:589:13: note: in instantiation of member function 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_grow_capacity' requested here
        TRY(try_grow_capacity(m_size + 1));
            ^
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:304:14: note: in instantiation of function template specialization 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_empend<void (GL::GLContext::*)(unsigned int, unsigned int), AK::Tuple<unsigned int, unsigned int>>' requested here
        MUST(try_empend(forward<Args>(args)...));
             ^
/home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:267:50: note: in instantiation of function template specialization 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::empend<void (GL::GLContext::*)(unsigned int, unsigned int), AK::Tuple<unsigned int, unsigned int>>' requested here
        m_current_listing_index->listing.entries.empend(member, Listing::ArgumentsFor<member> { forward<Args>(args)... });
                                                 ^
/home/leon/serenity/Userland/Libraries/LibGL/Lighting.cpp:96:5: note: in instantiation of function template specialization 'GL::GLContext::append_to_listing<&GL::GLContext::gl_color_material, unsigned int &, unsigned int &>' requested here
    APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED(gl_color_material, face, mode);
    ^
/home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:40:9: note: expanded from macro 'APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED'
        append_to_listing<&GLContext::name>(__VA_ARGS__);   \
        ^
1 error generated.
[6/11] Building CXX object Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Matrix.cpp.o
FAILED: Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Matrix.cpp.o 
/usr/bin/ccache /usr/bin/clang++ -DLibGL_EXPORTS -I/home/leon/serenity/Meta/Lagom/../.. -I/home/leon/serenity/Meta/Lagom/../../Userland -I/home/leon/serenity/Meta/Lagom/../../Userland/Libraries -I/home/leon/serenity/Meta/Lagom/../../Userland/Services -I/home/leon/serenity/Build/lagom -I/home/leon/serenity/Build/lagom/Userland/Libraries -I/home/leon/serenity/Build/lagom/Userland/Services -fPIC -Wall -Wextra -Werror -Wno-implicit-const-int-float-conversion -Wno-literal-suffix -Wno-maybe-uninitialized -Wno-unknown-warning-option -Wno-unused-command-line-argument -Wnogcc-compat -fsigned-char -fno-exceptions -fdiagnostics-color=always -g1 -O2 -fno-semantic-interposition -fPIC -Wno-overloaded-virtual -Wno-user-defined-literals -fconstexpr-steps=16777216 -Wno-unused-private-field -std=c++20 -MD -MT Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Matrix.cpp.o -MF Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Matrix.cpp.o.d -o Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Matrix.cpp.o -c /home/leon/serenity/Userland/Libraries/LibGL/Matrix.cpp
In file included from /home/leon/serenity/Userland/Libraries/LibGL/Matrix.cpp:10:
In file included from /home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:11:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/ByteBuffer.h:11:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Error.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/StringView.h:10:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Checked.h:31:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Concepts.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Forward.h:10:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/SinglyLinkedListSizePolicy.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Types.h:10:
/home/leon/serenity/Meta/Lagom/../../AK/StdLibExtras.h:75:12: error: cannot cast from lvalue of type 'const Gfx::Matrix<4, float>' to rvalue reference type 'Gfx::Matrix<4, float> &&'; types are not compatible
    return static_cast<T&&>(arg);
           ^~~~~~~~~~~~~~~~~~~~~
/home/leon/serenity/Meta/Lagom/../../AK/Tuple.h:184:32: note: in instantiation of function template specialization 'std::move<Gfx::Matrix<4, float>>' requested here
        : Detail::Tuple<Ts...>(move(other.get<Is>())...)
                               ^
/home/leon/serenity/Meta/Lagom/../../AK/Tuple.h:122:11: note: in instantiation of function template specialization 'AK::Tuple<const Gfx::Matrix<4, float> &>::Tuple<0U>' requested here
        : Tuple(move(other), Indices())
          ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:60:28: note: in instantiation of member function 'AK::Tuple<const Gfx::Matrix<4, float> &>::Tuple' requested here
            new (new_data) F(move(*bit_cast<F*>(old_data)));
                           ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: in instantiation of member function 'AK::Detail::Variant<unsigned char, '\t', AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<unsigned int>, AK::Tuple<double, double, double, double, double, double>, AK::Tuple<>, AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<float, float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<int, unsigned int, const void *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned char>, AK::Tuple<int, int, unsigned int, unsigned int, const void *>, AK::Tuple<double, double>, AK::Tuple<float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int, unsigned int, int, unsigned int>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, unsigned int, unsigned int>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<float>, AK::Tuple<unsigned int>, AK::Tuple<>, AK::Tuple<unsigned int, float, float, float, float>, AK::Tuple<int, int, float, float, float, float, const unsigned char *>, AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>, AK::Tuple<double, double, double, double>, AK::Tuple<unsigned int, unsigned int, Gfx::VectorN<4, float>>, AK::Tuple<unsigned int, unsigned int, int>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, const float *>, AK::Tuple<unsigned int, int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, void *, unsigned int>, AK::Tuple<unsigned int, const double *>, AK::Tuple<unsigned int, int, int, int, int, int, int, int>, AK::Tuple<float>>::move_' requested here
            Variant<IndexType, InitialIndex + 1, Ts...>::move_(old_id, old_data, new_data);
                                                         ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: in instantiation of member function 'AK::Detail::Variant<unsigned char, '\b', AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<unsigned int>, AK::Tuple<double, double, double, double, double, double>, AK::Tuple<>, AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<float, float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<int, unsigned int, const void *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned char>, AK::Tuple<int, int, unsigned int, unsigned int, const void *>, AK::Tuple<double, double>, AK::Tuple<float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int, unsigned int, int, unsigned int>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, unsigned int, unsigned int>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<float>, AK::Tuple<unsigned int>, AK::Tuple<>, AK::Tuple<unsigned int, float, float, float, float>, AK::Tuple<int, int, float, float, float, float, const unsigned char *>, AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>, AK::Tuple<double, double, double, double>, AK::Tuple<unsigned int, unsigned int, Gfx::VectorN<4, float>>, AK::Tuple<unsigned int, unsigned int, int>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, const float *>, AK::Tuple<unsigned int, int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, void *, unsigned int>, AK::Tuple<unsigned int, const double *>, AK::Tuple<unsigned int, int, int, int, int, int, int, int>, AK::Tuple<float>>::move_' requested here
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: (skipping 9 contexts in backtrace; use -ftemplate-backtrace-limit=0 to see all)
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:641:16: note: in instantiation of member function 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_ensure_capacity' requested here
        return try_ensure_capacity(padded_capacity(needed_capacity));
               ^
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:589:13: note: in instantiation of member function 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_grow_capacity' requested here
        TRY(try_grow_capacity(m_size + 1));
            ^
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:304:14: note: in instantiation of function template specialization 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_empend<void (GL::GLContext::*)(double, double, double, double, double, double), AK::Tuple<double, double, double, double, double, double>>' requested here
        MUST(try_empend(forward<Args>(args)...));
             ^
/home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:267:50: note: in instantiation of function template specialization 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::empend<void (GL::GLContext::*)(double, double, double, double, double, double), AK::Tuple<double, double, double, double, double, double>>' requested here
        m_current_listing_index->listing.entries.empend(member, Listing::ArgumentsFor<member> { forward<Args>(args)... });
                                                 ^
/home/leon/serenity/Userland/Libraries/LibGL/Matrix.cpp:29:5: note: in instantiation of function template specialization 'GL::GLContext::append_to_listing<&GL::GLContext::gl_frustum, double &, double &, double &, double &, double &, double &>' requested here
    APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED(gl_frustum, left, right, bottom, top, near_val, far_val);
    ^
/home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:40:9: note: expanded from macro 'APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED'
        append_to_listing<&GLContext::name>(__VA_ARGS__);   \
        ^
1 error generated.
[7/11] Building CXX object Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/GLContext.cpp.o
FAILED: Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/GLContext.cpp.o 
/usr/bin/ccache /usr/bin/clang++ -DLibGL_EXPORTS -I/home/leon/serenity/Meta/Lagom/../.. -I/home/leon/serenity/Meta/Lagom/../../Userland -I/home/leon/serenity/Meta/Lagom/../../Userland/Libraries -I/home/leon/serenity/Meta/Lagom/../../Userland/Services -I/home/leon/serenity/Build/lagom -I/home/leon/serenity/Build/lagom/Userland/Libraries -I/home/leon/serenity/Build/lagom/Userland/Services -fPIC -Wall -Wextra -Werror -Wno-implicit-const-int-float-conversion -Wno-literal-suffix -Wno-maybe-uninitialized -Wno-unknown-warning-option -Wno-unused-command-line-argument -Wnogcc-compat -fsigned-char -fno-exceptions -fdiagnostics-color=always -g1 -O2 -fno-semantic-interposition -fPIC -Wno-overloaded-virtual -Wno-user-defined-literals -fconstexpr-steps=16777216 -Wno-unused-private-field -std=c++20 -MD -MT Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/GLContext.cpp.o -MF Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/GLContext.cpp.o.d -o Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/GLContext.cpp.o -c /home/leon/serenity/Userland/Libraries/LibGL/GLContext.cpp
In file included from /home/leon/serenity/Userland/Libraries/LibGL/GLContext.cpp:10:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/StringBuilder.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/ByteBuffer.h:11:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Error.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/StringView.h:10:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Checked.h:31:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Concepts.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Forward.h:10:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/SinglyLinkedListSizePolicy.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Types.h:10:
/home/leon/serenity/Meta/Lagom/../../AK/StdLibExtras.h:75:12: error: cannot cast from lvalue of type 'const Gfx::Matrix<4, float>' to rvalue reference type 'Gfx::Matrix<4, float> &&'; types are not compatible
    return static_cast<T&&>(arg);
           ^~~~~~~~~~~~~~~~~~~~~
/home/leon/serenity/Meta/Lagom/../../AK/Tuple.h:184:32: note: in instantiation of function template specialization 'std::move<Gfx::Matrix<4, float>>' requested here
        : Detail::Tuple<Ts...>(move(other.get<Is>())...)
                               ^
/home/leon/serenity/Meta/Lagom/../../AK/Tuple.h:122:11: note: in instantiation of function template specialization 'AK::Tuple<const Gfx::Matrix<4, float> &>::Tuple<0U>' requested here
        : Tuple(move(other), Indices())
          ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:60:28: note: in instantiation of member function 'AK::Tuple<const Gfx::Matrix<4, float> &>::Tuple' requested here
            new (new_data) F(move(*bit_cast<F*>(old_data)));
                           ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: in instantiation of member function 'AK::Detail::Variant<unsigned char, '\t', AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<unsigned int>, AK::Tuple<double, double, double, double, double, double>, AK::Tuple<>, AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<float, float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<int, unsigned int, const void *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned char>, AK::Tuple<int, int, unsigned int, unsigned int, const void *>, AK::Tuple<double, double>, AK::Tuple<float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int, unsigned int, int, unsigned int>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, unsigned int, unsigned int>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<float>, AK::Tuple<unsigned int>, AK::Tuple<>, AK::Tuple<unsigned int, float, float, float, float>, AK::Tuple<int, int, float, float, float, float, const unsigned char *>, AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>, AK::Tuple<double, double, double, double>, AK::Tuple<unsigned int, unsigned int, Gfx::VectorN<4, float>>, AK::Tuple<unsigned int, unsigned int, int>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, const float *>, AK::Tuple<unsigned int, int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, void *, unsigned int>, AK::Tuple<unsigned int, const double *>, AK::Tuple<unsigned int, int, int, int, int, int, int, int>, AK::Tuple<float>>::move_' requested here
            Variant<IndexType, InitialIndex + 1, Ts...>::move_(old_id, old_data, new_data);
                                                         ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: in instantiation of member function 'AK::Detail::Variant<unsigned char, '\b', AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<unsigned int>, AK::Tuple<double, double, double, double, double, double>, AK::Tuple<>, AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<float, float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<int, unsigned int, const void *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned char>, AK::Tuple<int, int, unsigned int, unsigned int, const void *>, AK::Tuple<double, double>, AK::Tuple<float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int, unsigned int, int, unsigned int>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, unsigned int, unsigned int>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<float>, AK::Tuple<unsigned int>, AK::Tuple<>, AK::Tuple<unsigned int, float, float, float, float>, AK::Tuple<int, int, float, float, float, float, const unsigned char *>, AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>, AK::Tuple<double, double, double, double>, AK::Tuple<unsigned int, unsigned int, Gfx::VectorN<4, float>>, AK::Tuple<unsigned int, unsigned int, int>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, const float *>, AK::Tuple<unsigned int, int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, void *, unsigned int>, AK::Tuple<unsigned int, const double *>, AK::Tuple<unsigned int, int, int, int, int, int, int, int>, AK::Tuple<float>>::move_' requested here
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: (skipping 9 contexts in backtrace; use -ftemplate-backtrace-limit=0 to see all)
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:641:16: note: in instantiation of member function 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_ensure_capacity' requested here
        return try_ensure_capacity(padded_capacity(needed_capacity));
               ^
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:589:13: note: in instantiation of member function 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_grow_capacity' requested here
        TRY(try_grow_capacity(m_size + 1));
            ^
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:304:14: note: in instantiation of function template specialization 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_empend<void (GL::GLContext::*)(unsigned int), AK::Tuple<unsigned int>>' requested here
        MUST(try_empend(forward<Args>(args)...));
             ^
/home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:267:50: note: in instantiation of function template specialization 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::empend<void (GL::GLContext::*)(unsigned int), AK::Tuple<unsigned int>>' requested here
        m_current_listing_index->listing.entries.empend(member, Listing::ArgumentsFor<member> { forward<Args>(args)... });
                                                 ^
/home/leon/serenity/Userland/Libraries/LibGL/GLContext.cpp:85:5: note: in instantiation of function template specialization 'GL::GLContext::append_to_listing<&GL::GLContext::gl_begin, unsigned int &>' requested here
    APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED(gl_begin, mode);
    ^
/home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:40:9: note: expanded from macro 'APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED'
        append_to_listing<&GLContext::name>(__VA_ARGS__);   \
        ^
1 error generated.
[8/11] Building CXX object Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Texture.cpp.o
FAILED: Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Texture.cpp.o 
/usr/bin/ccache /usr/bin/clang++ -DLibGL_EXPORTS -I/home/leon/serenity/Meta/Lagom/../.. -I/home/leon/serenity/Meta/Lagom/../../Userland -I/home/leon/serenity/Meta/Lagom/../../Userland/Libraries -I/home/leon/serenity/Meta/Lagom/../../Userland/Services -I/home/leon/serenity/Build/lagom -I/home/leon/serenity/Build/lagom/Userland/Libraries -I/home/leon/serenity/Build/lagom/Userland/Services -fPIC -Wall -Wextra -Werror -Wno-implicit-const-int-float-conversion -Wno-literal-suffix -Wno-maybe-uninitialized -Wno-unknown-warning-option -Wno-unused-command-line-argument -Wnogcc-compat -fsigned-char -fno-exceptions -fdiagnostics-color=always -g1 -O2 -fno-semantic-interposition -fPIC -Wno-overloaded-virtual -Wno-user-defined-literals -fconstexpr-steps=16777216 -Wno-unused-private-field -std=c++20 -MD -MT Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Texture.cpp.o -MF Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Texture.cpp.o.d -o Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/Texture.cpp.o -c /home/leon/serenity/Userland/Libraries/LibGL/Texture.cpp
In file included from /home/leon/serenity/Userland/Libraries/LibGL/Texture.cpp:9:
In file included from /home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:11:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/ByteBuffer.h:11:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Error.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/StringView.h:10:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Checked.h:31:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Concepts.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Forward.h:10:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/SinglyLinkedListSizePolicy.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Types.h:10:
/home/leon/serenity/Meta/Lagom/../../AK/StdLibExtras.h:75:12: error: cannot cast from lvalue of type 'const Gfx::Matrix<4, float>' to rvalue reference type 'Gfx::Matrix<4, float> &&'; types are not compatible
    return static_cast<T&&>(arg);
           ^~~~~~~~~~~~~~~~~~~~~
/home/leon/serenity/Meta/Lagom/../../AK/Tuple.h:184:32: note: in instantiation of function template specialization 'std::move<Gfx::Matrix<4, float>>' requested here
        : Detail::Tuple<Ts...>(move(other.get<Is>())...)
                               ^
/home/leon/serenity/Meta/Lagom/../../AK/Tuple.h:122:11: note: in instantiation of function template specialization 'AK::Tuple<const Gfx::Matrix<4, float> &>::Tuple<0U>' requested here
        : Tuple(move(other), Indices())
          ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:60:28: note: in instantiation of member function 'AK::Tuple<const Gfx::Matrix<4, float> &>::Tuple' requested here
            new (new_data) F(move(*bit_cast<F*>(old_data)));
                           ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: in instantiation of member function 'AK::Detail::Variant<unsigned char, '\t', AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<unsigned int>, AK::Tuple<double, double, double, double, double, double>, AK::Tuple<>, AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<float, float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<int, unsigned int, const void *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned char>, AK::Tuple<int, int, unsigned int, unsigned int, const void *>, AK::Tuple<double, double>, AK::Tuple<float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int, unsigned int, int, unsigned int>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, unsigned int, unsigned int>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<float>, AK::Tuple<unsigned int>, AK::Tuple<>, AK::Tuple<unsigned int, float, float, float, float>, AK::Tuple<int, int, float, float, float, float, const unsigned char *>, AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>, AK::Tuple<double, double, double, double>, AK::Tuple<unsigned int, unsigned int, Gfx::VectorN<4, float>>, AK::Tuple<unsigned int, unsigned int, int>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, const float *>, AK::Tuple<unsigned int, int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, void *, unsigned int>, AK::Tuple<unsigned int, const double *>, AK::Tuple<unsigned int, int, int, int, int, int, int, int>, AK::Tuple<float>>::move_' requested here
            Variant<IndexType, InitialIndex + 1, Ts...>::move_(old_id, old_data, new_data);
                                                         ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: in instantiation of member function 'AK::Detail::Variant<unsigned char, '\b', AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<unsigned int>, AK::Tuple<double, double, double, double, double, double>, AK::Tuple<>, AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<float, float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<int, unsigned int, const void *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned char>, AK::Tuple<int, int, unsigned int, unsigned int, const void *>, AK::Tuple<double, double>, AK::Tuple<float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int, unsigned int, int, unsigned int>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, unsigned int, unsigned int>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<float>, AK::Tuple<unsigned int>, AK::Tuple<>, AK::Tuple<unsigned int, float, float, float, float>, AK::Tuple<int, int, float, float, float, float, const unsigned char *>, AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>, AK::Tuple<double, double, double, double>, AK::Tuple<unsigned int, unsigned int, Gfx::VectorN<4, float>>, AK::Tuple<unsigned int, unsigned int, int>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, const float *>, AK::Tuple<unsigned int, int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, void *, unsigned int>, AK::Tuple<unsigned int, const double *>, AK::Tuple<unsigned int, int, int, int, int, int, int, int>, AK::Tuple<float>>::move_' requested here
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: (skipping 9 contexts in backtrace; use -ftemplate-backtrace-limit=0 to see all)
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:641:16: note: in instantiation of member function 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_ensure_capacity' requested here
        return try_ensure_capacity(padded_capacity(needed_capacity));
               ^
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:589:13: note: in instantiation of member function 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_grow_capacity' requested here
        TRY(try_grow_capacity(m_size + 1));
            ^
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:304:14: note: in instantiation of function template specialization 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_empend<void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>>' requested here
        MUST(try_empend(forward<Args>(args)...));
             ^
/home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:267:50: note: in instantiation of function template specialization 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::empend<void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>>' requested here
        m_current_listing_index->listing.entries.empend(member, Listing::ArgumentsFor<member> { forward<Args>(args)... });
                                                 ^
/home/leon/serenity/Userland/Libraries/LibGL/Texture.cpp:97:5: note: in instantiation of function template specialization 'GL::GLContext::append_to_listing<&GL::GLContext::gl_copy_tex_image_2d, unsigned int &, int &, unsigned int &, int &, int &, int &, int &, int &>' requested here
    APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED(gl_copy_tex_image_2d, target, level, internalformat, x, y, width, height, border);
    ^
/home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:40:9: note: expanded from macro 'APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED'
        append_to_listing<&GLContext::name>(__VA_ARGS__);   \
        ^
1 error generated.
[9/11] Building CXX object Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/List.cpp.o
FAILED: Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/List.cpp.o 
/usr/bin/ccache /usr/bin/clang++ -DLibGL_EXPORTS -I/home/leon/serenity/Meta/Lagom/../.. -I/home/leon/serenity/Meta/Lagom/../../Userland -I/home/leon/serenity/Meta/Lagom/../../Userland/Libraries -I/home/leon/serenity/Meta/Lagom/../../Userland/Services -I/home/leon/serenity/Build/lagom -I/home/leon/serenity/Build/lagom/Userland/Libraries -I/home/leon/serenity/Build/lagom/Userland/Services -fPIC -Wall -Wextra -Werror -Wno-implicit-const-int-float-conversion -Wno-literal-suffix -Wno-maybe-uninitialized -Wno-unknown-warning-option -Wno-unused-command-line-argument -Wnogcc-compat -fsigned-char -fno-exceptions -fdiagnostics-color=always -g1 -O2 -fno-semantic-interposition -fPIC -Wno-overloaded-virtual -Wno-user-defined-literals -fconstexpr-steps=16777216 -Wno-unused-private-field -std=c++20 -MD -MT Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/List.cpp.o -MF Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/List.cpp.o.d -o Userland/Libraries/LibGL/CMakeFiles/LibGL.dir/List.cpp.o -c /home/leon/serenity/Userland/Libraries/LibGL/List.cpp
In file included from /home/leon/serenity/Userland/Libraries/LibGL/List.cpp:10:
In file included from /home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:11:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/ByteBuffer.h:11:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Error.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/StringView.h:10:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Checked.h:31:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Concepts.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Forward.h:10:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/SinglyLinkedListSizePolicy.h:9:
In file included from /home/leon/serenity/Meta/Lagom/../../AK/Types.h:10:
/home/leon/serenity/Meta/Lagom/../../AK/StdLibExtras.h:75:12: error: cannot cast from lvalue of type 'const Gfx::Matrix<4, float>' to rvalue reference type 'Gfx::Matrix<4, float> &&'; types are not compatible
    return static_cast<T&&>(arg);
           ^~~~~~~~~~~~~~~~~~~~~
/home/leon/serenity/Meta/Lagom/../../AK/Tuple.h:184:32: note: in instantiation of function template specialization 'std::move<Gfx::Matrix<4, float>>' requested here
        : Detail::Tuple<Ts...>(move(other.get<Is>())...)
                               ^
/home/leon/serenity/Meta/Lagom/../../AK/Tuple.h:122:11: note: in instantiation of function template specialization 'AK::Tuple<const Gfx::Matrix<4, float> &>::Tuple<0U>' requested here
        : Tuple(move(other), Indices())
          ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:60:28: note: in instantiation of member function 'AK::Tuple<const Gfx::Matrix<4, float> &>::Tuple' requested here
            new (new_data) F(move(*bit_cast<F*>(old_data)));
                           ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: in instantiation of member function 'AK::Detail::Variant<unsigned char, '\t', AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<unsigned int>, AK::Tuple<double, double, double, double, double, double>, AK::Tuple<>, AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<float, float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<int, unsigned int, const void *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned char>, AK::Tuple<int, int, unsigned int, unsigned int, const void *>, AK::Tuple<double, double>, AK::Tuple<float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int, unsigned int, int, unsigned int>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, unsigned int, unsigned int>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<float>, AK::Tuple<unsigned int>, AK::Tuple<>, AK::Tuple<unsigned int, float, float, float, float>, AK::Tuple<int, int, float, float, float, float, const unsigned char *>, AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>, AK::Tuple<double, double, double, double>, AK::Tuple<unsigned int, unsigned int, Gfx::VectorN<4, float>>, AK::Tuple<unsigned int, unsigned int, int>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, const float *>, AK::Tuple<unsigned int, int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, void *, unsigned int>, AK::Tuple<unsigned int, const double *>, AK::Tuple<unsigned int, int, int, int, int, int, int, int>, AK::Tuple<float>>::move_' requested here
            Variant<IndexType, InitialIndex + 1, Ts...>::move_(old_id, old_data, new_data);
                                                         ^
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: in instantiation of member function 'AK::Detail::Variant<unsigned char, '\b', AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<unsigned int>, AK::Tuple<double, double, double, double, double, double>, AK::Tuple<>, AK::Tuple<>, AK::Tuple<const Gfx::Matrix<4, float> &>, AK::Tuple<float, float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<int, unsigned int, const void *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned char>, AK::Tuple<int, int, unsigned int, unsigned int, const void *>, AK::Tuple<double, double>, AK::Tuple<float, float>, AK::Tuple<int, int, int, int>, AK::Tuple<unsigned int, unsigned int, int, unsigned int>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, unsigned int, unsigned int>, AK::Tuple<float, float, float>, AK::Tuple<float, float, float, float>, AK::Tuple<float>, AK::Tuple<unsigned int>, AK::Tuple<>, AK::Tuple<unsigned int, float, float, float, float>, AK::Tuple<int, int, float, float, float, float, const unsigned char *>, AK::Tuple<unsigned int, int, unsigned int, int, int, int, int, int>, AK::Tuple<double, double, double, double>, AK::Tuple<unsigned int, unsigned int, Gfx::VectorN<4, float>>, AK::Tuple<unsigned int, unsigned int, int>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, float>, AK::Tuple<unsigned int, const float *>, AK::Tuple<unsigned int, int>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int, float>, AK::Tuple<unsigned int, unsigned int, const float *>, AK::Tuple<unsigned int, unsigned int, const int *>, AK::Tuple<unsigned int, unsigned int>, AK::Tuple<unsigned int, unsigned int, void *, unsigned int>, AK::Tuple<unsigned int, const double *>, AK::Tuple<unsigned int, int, int, int, int, int, int, int>, AK::Tuple<float>>::move_' requested here
/home/leon/serenity/Meta/Lagom/../../AK/Variant.h:62:58: note: (skipping 9 contexts in backtrace; use -ftemplate-backtrace-limit=0 to see all)
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:641:16: note: in instantiation of member function 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_ensure_capacity' requested here
        return try_ensure_capacity(padded_capacity(needed_capacity));
               ^
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:589:13: note: in instantiation of member function 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_grow_capacity' requested here
        TRY(try_grow_capacity(m_size + 1));
            ^
/home/leon/serenity/Meta/Lagom/../../AK/Vector.h:304:14: note: in instantiation of function template specialization 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::try_empend<void (GL::GLContext::*)(unsigned int), AK::Tuple<unsigned int>>' requested here
        MUST(try_empend(forward<Args>(args)...));
             ^
/home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:267:50: note: in instantiation of function template specialization 'AK::Vector<GL::GLContext::Listing::FunctionAndArgs<void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(int), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(double, double, double, double, double, double), void (GL::GLContext::*)(), void (GL::GLContext::*)(), void (GL::GLContext::*)(const Gfx::Matrix<4, float> &), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(int, unsigned int, const void *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned char), void (GL::GLContext::*)(int, int, unsigned int, unsigned int, const void *), void (GL::GLContext::*)(double, double), void (GL::GLContext::*)(float, float), void (GL::GLContext::*)(int, int, int, int), void (GL::GLContext::*)(unsigned int, unsigned int, int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, unsigned int, unsigned int), void (GL::GLContext::*)(float, float, float), void (GL::GLContext::*)(float, float, float, float), void (GL::GLContext::*)(float), void (GL::GLContext::*)(unsigned int), void (GL::GLContext::*)(), void (GL::GLContext::*)(unsigned int, float, float, float, float), void (GL::GLContext::*)(int, int, float, float, float, float, const unsigned char *), void (GL::GLContext::*)(unsigned int, int, unsigned int, int, int, int, int, int), void (GL::GLContext::*)(double, double, double, double), void (GL::GLContext::*)(unsigned int, unsigned int, Gfx::VectorN<4, float>), void (GL::GLContext::*)(unsigned int, unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, float), void (GL::GLContext::*)(unsigned int, const float *), void (GL::GLContext::*)(unsigned int, int), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int, float), void (GL::GLContext::*)(unsigned int, unsigned int, const float *), void (GL::GLContext::*)(unsigned int, unsigned int, const int *), void (GL::GLContext::*)(unsigned int, unsigned int), void (GL::GLContext::*)(unsigned int, unsigned int, void *, unsigned int), void (GL::GLContext::*)(unsigned int, const double *), void (GL::GLContext::*)(unsigned int, int, int, int, int, int, int, int), void (GL::GLContext::*)(float)>, 0>::empend<void (GL::GLContext::*)(unsigned int), AK::Tuple<unsigned int>>' requested here
        m_current_listing_index->listing.entries.empend(member, Listing::ArgumentsFor<member> { forward<Args>(args)... });
                                                 ^
/home/leon/serenity/Userland/Libraries/LibGL/List.cpp:19:5: note: in instantiation of function template specialization 'GL::GLContext::append_to_listing<&GL::GLContext::gl_call_list, unsigned int &>' requested here
    APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED(gl_call_list, list);
    ^
/home/leon/serenity/Meta/Lagom/../../Userland/Libraries/LibGL/GLContext.h:40:9: note: expanded from macro 'APPEND_TO_CALL_LIST_AND_RETURN_IF_NEEDED'
        append_to_listing<&GLContext::name>(__VA_ARGS__);   \
        ^
1 error generated.
ninja: build stopped: subcommand failed.
```

</details>